src/urh/dev/native/lib/bladerf.pyx

```diff
@@ -129,8 +129,23 @@ cpdef int set_center_freq(bladerf_frequency frequency):
     return bladerf_set_frequency(_c_device, get_current_bladerf_channel(), frequency)
 
 cpdef bladerf_frequency get_center_freq():
-    cdef bladerf_frequency result = 0
-    err = bladerf_get_frequency(_c_device, get_current_bladerf_channel(), &result)
+  cdef bladerf_frequency result = 0
+  err = bladerf_get_frequency(_c_device, get_current_bladerf_channel(), &result)
+
+  if err != 0:
+      return 0
+
+  return result
+
+cpdef int set_bias_tee(on_or_off):
+  IF BLADERF_API_VERSION >= 2:
+    cdef bool bias_tee = 1 if on_or_off else 0
+    return bladerf_set_bias_tee(_c_device, get_current_bladerf_channel(), bias_tee)
+
+cpdef int get_bias_tee():
+  IF BLADERF_API_VERSION >= 2:
+    cdef bool result = 0
+    err = bladerf_get_bias_tee(_c_device, get_current_bladerf_channel(), &result)
 
     if err != 0:
         return 0
```

Comment:
Isn't that misleading? 0 seems to be a valid value for bias_tee and should not be returned in error case.

Reply:
You're right. I made the modification to catch the errors and return right value.